components/form-builder/app/responses/Dialogs/line-item-entries/LineItemEntries.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useEffect, useRef } from "react";
 import { LineItems } from "./LineItems";
```

Comment:
Moved the LinItem and related into the Dialogs directory since they are specific to the Dialogs and not really reusable.